terra/entity.json

```diff
@@ -28,248 +13,84 @@
     "website": "https://axelar.network/",
     "twitter": "https://twitter.com/axelarcore"
   },
-  {
-    "name": "Binance"
-  },
-  {
-    "name": "Bithumb"
-  },
   {
     "name": "boneLuna"
   },
-  {
-    "name": "Builder Council"
-  },
-  {
-    "name": "Capapult"
-  },
-  {
-    "name": "Coinex"
-  },
-  {
-    "name": "Coinhall"
-  },
-  {
-    "name": "Coinone"
-  },
-  {
-    "name": "Core Distribution Module"
-  },
-  {
-    "name": "Core Staking Module"
-  },
   {
     "name": "Cosmos",
     "website": "https://cosmos.network/",
     "telegram": "https://t.me/cosmosproject",
-    "twitter": "https://twitter.com/cosmos",
-    "coinmarketcap": "https://coinmarketcap.com/currencies/cosmos/",
-    "coingecko": "https://www.coingecko.com/en/coins/cosmos-hub"
+    "twitter": "https://twitter.com/cosmos"
   },
   {
     "name": "Crescent Network",
     "website": "crescent.network",
     "telegram": "http://t.me/crescentnetwork",
-    "twitter": "https://twitter.com/crescenthub",
-    "coingecko": "https://www.coingecko.com/en/coins/crescent-network"
-  },
-  {
-    "name": "Cyberdeck"
-  },
-  {
-    "name": "Edge"
-  },
-  {
-    "name": "Edge Protocol"
+    "twitter": "https://twitter.com/crescenthub"
   },
   {
     "name": "Eris Protocol",
     "website": "https://www.erisprotocol.com/",
     "telegram": "https://t.co/cqfIpelqbk",
     "twitter": "https://twitter.com/eris_protocol"
   },
-  {
-    "name": "FTX"
-  },
-  {
-    "name": "Galactic Punks"
-  },
   {
     "name": "Gidorah"
   },
-  {
-    "name": "Gravidao"
-  },
   {
     "name": "Gravity Bridge",
     "website": "https://t.co/rSb0msPbRI",
-    "twitter": "https://twitter.com/gravity_bridge",
-    "coinmarketcap": "https://coinmarketcap.com/currencies/grav/",
```

Comment:
Did this cmc and coingecko link belong to `ibc/64B3EFE08A6A3EF5D2A7F1D3AF033DDC951DA51F92E06CC31D3F9AF8E8368860`? If so, we should add back.

Reply:
thanks for highlighting this! turns out I forgot to update the CI script -- fixed in the latest commit